_episodes/07-docker-image-examples.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "Examples of Using Container Images in Practice"
+teaching: 20
+exercises: 0
+questions:
+- "How can I use Docker for my own work?"
+objectives:
+- "Use existing container images and Docker in a research project."
+keypoints:
+- "There are many ways you might use Docker and existing container images in your research project."
+---
+
+Now that we have learned the basics of working with docker images and containers,
+let's apply what we learned to an example workflow.
+
+You may choose one or more of the following examples to practice using containers.
+
+## Jekyll Website Example
+
+In this [Jekyll Website example](../_extras/e02-jekyll-lesson-example), you can practice
+rendering this lesson website on your computer using Jekyll in a Docker container.
+Rendering the website in a container avoids a complicated software installation; instead of installing Jekyll and all the other tools needed to create the final website, all the work can be done in the container.
+
+
+## GitHub Actions Example
+
+In this [GitHub Actions example](../_extras/e01-github-actions), you can learn more about
+continuous integration in the cloud and how you can use container images with GitHub to
+automate repetitive tasks like testing code or deploying websites.
```

Comment:
Thought crosses my mind: will learners be thrown by us giving the Jekyll example above, and then mentioning deploying websites in the GitHub Actions example? Could possibly explicitly connect this, by noting that GitHub Actions could itself do the deployment mentioned in the Jekyll example, but ... that feels confusing...

Reply:
I had the same thought. Maybe make this into an issue that we can revisit later?

Reply:
I've been thinking of them as separate examples that instructors or learners could pick and choose from.  While these two examples could certainly be combined, I'm hoping for more examples that will be more unrelated.  So I hesitate to add connections on this page.  However, maybe we could add a call out at the bottom of the gh-actions one or both that days you could combine them? What do you think @dme26

Reply:
oh and @ChristinaLK? I didn't see your comment when I made mine for some reason.

Reply:
I agree that they are separate and should be thought of as separately, but there are some parallels.

Reply:
I'd leave it as is for now.